integration/spark/app/integrations/container/pysparkTruncateTableStartEvent.json

```diff
@@ -2,7 +2,7 @@
   "eventType": "START",
   "job": {
     "namespace": "testTruncateTable",
-    "name": "open_lineage_integration_truncate_table.execute_truncate_table_command"
+    "name": "open_lineage_integration_truncate_table_execute_truncate_table_command_truncate_test_truncate_table_test"
```

Comment:
Why not leave dots? I think they would help visually separate different parts of the job name.

Reply:
The reasoning behind that was for me: _let's do that same way as we already did for databricks runtime_, where we got rid of `dots`. Changing this would mean different behaviour for the databricks users during the upgrade with no option to mimic the current behaviour.  But dots look better to me ;)

@mobuchowski Shall we go for it?